Formula/ispc.rb

```diff
@@ -17,13 +17,26 @@ class Ispc < Formula
   depends_on "cmake" => :build
   depends_on "flex" => :build
   depends_on "python@3.9" => :build
-  depends_on "llvm@12"
+  depends_on "llvm"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
 
   def llvm
     deps.map(&:to_formula).find { |f| f.name.match? "^llvm" }
   end
 
   def install
+    # Force cmake to use our compiler shims instead of bypassing them.
+    inreplace "CMakeLists.txt", "set(CMAKE_C_COMPILER \"clang\")", "set(CMAKE_C_COMPILER \"#{ENV.cc}\")"
+    inreplace "CMakeLists.txt", "set(CMAKE_CXX_COMPILER \"clang++\")", "set(CMAKE_CXX_COMPILER \"#{ENV.cxx}\")"
```

Comment:
Can you make an issue upstream for this? I'd like to see your ideas confirmed or denied.

Reply:
Done: https://github.com/ispc/ispc/issues/2287.  There seems to be conflicting information in the upstream repo about whether building with GCC is okay, so I'm hoping the upstream developers can clarify this.  I do think we may still end up wanting to have this replacement even if we end up having to build with `clang` on Linux, because shim bypasses are something we should try to avoid whenever possible.